src/daft-scan/src/python.rs

```diff
@@ -430,7 +427,7 @@ pub mod pylib {
                 vec![data_source],
                 file_format.into(),
                 schema.schema,
-                storage_config.into(),
+                Arc::new(Default::default()), // read SQL doesn't actually use the storage config
```

Comment:
I think we should probably propagate this through since we already have the plumbing for it -- I can see us needing the config in the future (e.g. a `SQLConfig`) and it would be bad if the ScanTasks we create didn't have the correct config on hand haha

Reply:
Ok sounds good, I'll revert these changes